src/features/debugger/debugMessage.ts

```diff
@@ -236,6 +236,17 @@ async function checkMessage(
   } catch (err: any) {
     logger.info('Estimate gas call failed');
 
+    const errorReason = extractReasonString(err);
+    logger.debug(errorReason);
+
+    if (errorReason.includes(`execution reverted:`)) {
+      return {
```

Comment:
Wouldn't this cause us to miss out on more specific reasons for the bytecode and ICA cases just below

Reply:
yeah great catch
probably move this into the no handle block?